index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+      <div class="nav">
+        <header>
+            <div><a href="#" class="logo"><h1>Welcome</h1></a></div>
+        </header>
+        <nav>
+                
+            <div class="Togglebutton"><a href="#"><img class="MenuIcons" src="images/icons/icon-Menu.png" alt="togglebutton"></a></div>
```

Comment:
- [ ] As also suggested by the last reviewer, the `Togglebutton` is yet to load properly on the browser.

![image](https://user-images.githubusercontent.com/51296741/143586349-3ae4f0bf-9a3d-4f49-8e86-30010cefaf0a.png)

Change the current path to `src="images/Icons/Icon-Menu.png"` so it can properly grab the image.